openIMIS/openIMIS/settings.py

```diff
@@ -167,6 +164,7 @@ def SITE_URL():
     "channels",  # Websocket support
 ]
 INSTALLED_APPS += OPENIMIS_APPS
+INSTALLED_APPS += ["developer_tools"]
```

Comment:
I think it should be installed before other imis modules, not after.

Reply:
ok